TTS/api.py

```diff
@@ -0,0 +1,177 @@
+from pathlib import Path
+from TTS.utils.synthesizer import Synthesizer
+from TTS.utils.manage import ModelManager
+
+
+class TTS():
+    """TODO: Add voice conversion and Capacitron support.
+    """
+    def __init__(self, model_name:str=None, progress_bar:bool=True, gpu=False):
+        """🐸TTS python interface that allows to load and use the released models.
+
+        Args:
+            model_name (str, optional): Model name to load. You can list models by ```tts.models```. Defaults to None.
+            progress_bar (bool, optional): Whether to pring a progress bar while downloading a model. Defaults to True.
+            gpu (bool, optional): Enable/disable GPU. Some models might be too slow on CPU. Defaults to False.
+        """
+        self.manager = ModelManager(models_file=self.get_models_file_path(), progress_bar=progress_bar, verbose=False)
+        self.synthesizer = None
+        if model_name:
+            self.load_model_by_name(model_name, gpu)
+
+    @property
+    def models(self):
```

Comment:
I don't know a good solution, but it would be nice to be able to have that list of model before instantiating that class with the model name.

Reply:
We can keep the models in a python file instead of JSON and we can list it.

However can you give a code example for your intended use?

Reply:
Or just read the JSON

Reply:
This is already what `tts.models` does. Are you suggesting something different?

Reply:
But it works only when the class is already initialized because it uses a model manager instance. But this could be avoided if models was a `@staticmethod` instead.